### ISSUE TITLE
Cut routerLink

### ISSUE THREAD

**Issue by mezhik91** [akveo/ng2-smart-table#35]
I'm trying to add link with routerLink attribute into cell but routerLink is cutted. type is 'html'

**Comment by mansonkibe** [comment]
Am having the same issue too. I can't add  routerLink or any angular component.

**Comment by alan12111** [comment]
Same issue as well.

**Comment by FJB-ZA** [comment]
Did anyone find an answer for this issue yet?

**Comment by drchaks83** [comment]
Did anyone find an answer for this issue?

**Comment by mezhik91** [comment]
@drchaks83 yes, I'm just using Material Design tables https://material.angular.io/components/table/overview

**Comment by gyab12** [comment]
https://github.com/akveo/ng2-smart-table/issues/968#issuecomment-497265953
